tests/spec/core/markdown-spec.js

```diff
@@ -8,14 +8,18 @@ describe("Core - Markdown", function() {
     var ops = {
       config: makeBasicConfig(),
       body: makeDefaultBody() +
-       "\nFoo\n===\n",
+       "\n\nFoo\n===\n",
```

Comment:
That's weird. Not how CommonMark works, see http://spec.commonmark.org/dingus/?text=%0A%0AFoo%0A%3D%3D%3D%0A and http://spec.commonmark.org/dingus/?text=Foo%0A%3D%3D%3D%0A.